src/Entity/Event.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\EventRepository")
+ */
+class Event
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $shortDescription;
+
+    /**
+     * @ORM\Column(type="string",nullable=true)
+     */
+    private $longDescription;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $beginningDate;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $endDate;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $location;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Country", inversedBy="events")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $country;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $priceMin;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $priceMax;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $priceCurrency;
+
+    /**
+     * @ORM\Column(type="string", length=255)
```

Comment:
@daria-sieroshtan 
I think you may want to have this as entity, ie can be clickable to find out other events from save gang

Reply:
Not sure, it is expected to be free-form description, bunch of names, for example.